Polkadex-Substrate/Polkadex-Orderbook-Frontend
# feat: integrate cex on-ramp cede.store widget

## Description
Following our discussion regarding the CEX On-Ramp widget integration with cede.store, I integrate the cede.store widget allowing to transfer funds from centralized exchanges to a wallet connetected to Polkadex.

## Changes Made

Added a widget page
<img width="1511" alt="image" src="https://github.com/Polkadex-Substrate/Polkadex-Orderbook-Frontend/assets/57397713/6ee24075-ff6c-49a1-8382-47243f2db93e">

Added a sidebar link to on-ramp widget page:
<img width="476" alt="image" src="https://github.com/Polkadex-Substrate/Polkadex-Orderbook-Frontend/assets/57397713/190f6dd7-6cde-42f4-bfbc-bac77cca7ecd">

Added an on-ramp link to the modal:
<img width="384" alt="image" src="https://github.com/Polkadex-Substrate/Polkadex-Orderbook-Frontend/assets/57397713/ec3113a1-db34-41f3-a382-1998da971cce">


## How to Test

Check this page: http://localhost:3000/cexOnRamp
You can downoad cede.store extension, add a cex and try to withdraw. If you don't want to import your API keys or don't have a CEX account, you can use the extension in the demo mode to check the withdrawal.

## Checklist



- [ ] Included link to corresponding [Polkadex Orderbook Frontend Issue](https://github.com/Polkadex-Substrate/Polkadex-Orderbook-Frontend/issues).
- [ ] I have requested a review from at least one other contributor.
